--- tesseract-3.05.00/ccstruct/imagedata.cpp	2017-02-16 18:59:48.000000000 +0100
+++ tesseract-3.05.00-new/ccstruct/imagedata.cpp	2018-02-07 17:40:54.828144350 +0100
@@ -30,11 +30,13 @@
 #include "helpers.h"
 #include "tprintf.h"
 
-#if defined(__MINGW32__)
+//#if defined(__MINGW32__)
 # include <unistd.h>
+/*
 #elif __cplusplus > 199711L   // in C++11
 # include <thread>
 #endif
+*/
 
 // Number of documents to read ahead while training. Doesn't need to be very
 // large.
@@ -453,7 +455,8 @@
     if (needs_loading) LoadPageInBackground(index);
     // We can't directly load the page, or the background load will delete it
     // while the caller is using it, so give it a chance to work.
-#if __cplusplus > 199711L
+/*#if __cplusplus > 199711L*/
+#if 0
     std::this_thread::sleep_for(std::chrono::seconds(1));
 #elif _WIN32  // MSVS
     Sleep(1000);
