--- glew-1.13.0/Makefile	2015-08-10 13:47:22.000000000 +0200
+++ glew-1.13.0-win-64/Makefile	2019-02-19 20:26:46.092586659 +0100
@@ -77,14 +77,14 @@
 OPT = $(POPT)
 endif
 INCLUDE = -Iinclude
-CFLAGS = $(OPT) $(WARN) $(INCLUDE) $(CFLAGS.EXTRA)
+CFLAGS = $(OPT) $(WARN) $(INCLUDE) $(CFLAGS.EXTRA) $(CFLAGS.CMDLINE)
 
 all debug: glew.lib glew.lib.mx glew.bin
 
 # GLEW shared and static libraries
 
-LIB.LDFLAGS        := $(LDFLAGS.EXTRA) $(LDFLAGS.GL)
-LIB.LIBS           := $(GL_LDFLAGS)
+LIB.LDFLAGS        := $(LDFLAGS.EXTRA) $(LDFLAGS.GL) $(LDFLAGS.CMDLINE)
+LIB.LIBS           := $(GL_LDFLAGS) $(LIBS.CMDLINE)
 
 LIB.SRCS           := src/glew.c
 LIB.SRCS.NAMES     := $(notdir $(LIB.SRCS))
