--- glew-1.13.0-linux-64/Makefile	2019-03-29 22:51:30.545835035 +0100
+++ glew-1.13.0-win-64/Makefile	2019-03-29 22:55:06.832978418 +0100
@@ -115,14 +115,11 @@
 endif
 
 lib/$(LIB.SHARED): $(LIB.SOBJS)
-	$(LD) $(LDFLAGS.SO) -o $@ $^ $(LIB.LDFLAGS) $(LIB.LIBS)
+	touch $@
 ifneq ($(LN),)
 	$(LN) $(LIB.SHARED) lib/$(LIB.SONAME)
 	$(LN) $(LIB.SHARED) lib/$(LIB.DEVLNK)
 endif
-ifneq ($(STRIP),)
-	$(STRIP) -x $@
-endif
 
 tmp/$(SYSTEM)/default/static/glew.o: src/glew.c include/GL/glew.h include/GL/wglew.h include/GL/glxew.h
 	@mkdir -p $(dir $@)
@@ -163,14 +160,11 @@
 endif
 
 lib/$(LIB.SHARED.MX): $(LIB.SOBJS.MX)
-	$(LD) $(LDFLAGS.SO.MX) -o $@ $^ $(LIB.LDFLAGS) $(LIB.LIBS)
+	touch $@
 ifneq ($(LN),)
 	$(LN) $(LIB.SHARED.MX) lib/$(LIB.SONAME.MX)
 	$(LN) $(LIB.SHARED.MX) lib/$(LIB.DEVLNK.MX)
 endif
-ifneq ($(STRIP),)
-	$(STRIP) -x $@
-endif
 
 tmp/$(SYSTEM)/mx/static/glew.o: src/glew.c include/GL/glew.h include/GL/wglew.h include/GL/glxew.h
 	@mkdir -p $(dir $@)
