--- openal-soft-1.19.1/config.h.in	2018-10-12 00:05:31.000000000 +0200
+++ openal-soft-1.19.1-mac-64-new/config.h.in	2021-01-31 04:38:36.000000000 +0100
@@ -1,6 +1,6 @@
 /* API declaration export attribute */
-#define AL_API  ${EXPORT_DECL}
-#define ALC_API ${EXPORT_DECL}
+#define AL_API  __attribute__((visibility("hidden")))
+#define ALC_API __attribute__((visibility("hidden")))
 
 /* Define any available alignment declaration */
 #define ALIGN(x) ${ALIGN_DECL}
