--- opusfile-0.7-old/include/opusfile.h	2015-12-31 19:29:53.000000000 +0100
+++ opusfile-0.7/include/opusfile.h	2017-07-30 15:20:36.000000000 +0200
@@ -121,9 +121,9 @@
 #  endif
 # endif
 
-# if OP_GNUC_PREREQ(4,0)
-#  pragma GCC visibility push(default)
-# endif
+/*# if OP_GNUC_PREREQ(4,0)*/
+/*#  pragma GCC visibility push(default)*/
+/*# endif*/
 
 typedef struct OpusHead          OpusHead;
 typedef struct OpusTags          OpusTags;
@@ -2137,9 +2137,9 @@
 /*@}*/
 /*@}*/
 
-# if OP_GNUC_PREREQ(4,0)
-#  pragma GCC visibility pop
-# endif
+/*# if OP_GNUC_PREREQ(4,0)*/
+/*#  pragma GCC visibility pop*/
+/*# endif*/
 
 # if defined(__cplusplus)
 }
